### PR TITLE
Implemented new Biolo skills

### PR DESCRIPTION
**Pull request by Haydrich**
* **Addressed Issue(s)**: 

* **Server Mode**: RE

* **Description of Pull Request**: 
This is the initial release of the new biolo skills, [Fixed casting time/Variable Casting time/Cooldown/Global Cool Time] are dummies until providing a new information relative.
The new status information still unknown, So the flags are dummy too
I followed the information of 
https://ro.gnjoy.com/news/devnote/View.asp?category=1&seq=4184991&curpage=1
Translated: https://www.divine-pride.net/forum/index.php?/topic/7814-kro-27560-level-expansion-and-new-skills/
PS: This skills require new clientdate above 2024/08 depending on the release date from Ragnarok.

### CODE REVIEW

File: db/re/skill_db.yml [hunk 1]
```diff
@@ -43010,6 +43010,73 @@ Body:
...
+      - Level: 2
+        Area: 2
+      - Level: 3
+        Area: 2
+      - Level: 4
+        Area: 3
+      - Level: 5
+        Area: 3
+    Duration1: 60000
+    Cooldown: 1500
```
Comment (Atemo) [inline db/re/skill_db.yml:43037]:
```suggestion
    AfterCastActDelay: 500
    Cooldown: 1500
```

File: db/re/skill_db.yml [hunk 2]
```diff
@@ -43010,6 +43010,73 @@ Body:
...
+      - Level: 2
+        Area: 2
+      - Level: 3
+        Area: 2
+      - Level: 4
+        Area: 3
+      - Level: 5
+        Area: 3
+    GiveAp: 2
+    Cooldown: 1500
```
Comment (Atemo) [inline db/re/skill_db.yml:43065]:
```suggestion
    AfterCastActDelay: 500
    Cooldown: 500
```

**Review by Lemongrass3110 — APPROVED** [review]

File: db/re/skill_db.yml [hunk 3]
```diff
@@ -43267,6 +43267,75 @@ Body:
...
+    Name: BO_DUST_EXPLOSION
+    Description: Dust Explosion
+    MaxLevel: 5
+    Type: Weapon
+    TargetType: Attack
+    DamageFlags:
+      Splash: true
+    Range: 3
+    Hit: Multi_Hit
+    HitCount: -5
```
Comment (Atemo) [inline db/re/skill_db.yml:43309]:
```suggestion
    HitCount: 5
```
Damage is increased by hits

File: db/re/status.yml [hunk 4]
```diff
@@ -9132,3 +9132,12 @@ Body:
       NoDispell: true
       NoBanishingBuster: true
       NoClearance: true
+  - Status: Mystery_Powder
+    Icon: EFST_MYSTERY_POWDER
+    DurationLookup: BO_MYSTERY_POWDER
+    Flags:
+      BlEffect: true
```
Comment (Atemo) [inline db/re/status.yml:9139]: Not required
```suggestion
```
Unknown if `DisplayPc` flag is set, if anyone have this info

Comment (Haydrich) [reply db/re/status.yml:9139]: Deleting this gonna remove the status effect

**Review by Atemo — APPROVED** [review]

**Comment by rainer8989** [comment]
Why is it need 2024/08 ragexe? At 2024/06 cant learn skills because of exe?

**Pull request merged**